seekers/game/camp.py

```diff
@@ -0,0 +1,24 @@
+import dataclasses
+
+from seekers.vector import Vector
+
+
+@dataclasses.dataclass
+class Camp:
+    id: str
+    owner: object
```

Comment:
This is EVIL!!

Reply:
This is one of the very few changes that I needed to made to prevent circulate imports. I hope this is at least a acceptable solution.